data/links.json

```diff
@@ -79,6 +79,11 @@
         "href": "https://segfault.csh.rit.edu/",
         "icon": "material:layers"
       }
```

Comment:
that's how it was when i forked it, seems to be the case with all of the last items